feature-group-definitions/promise-withresolvers.yml

```diff
@@ -0,0 +1,16 @@
+name: Promise.withResolvers()
+description: Promise.withResolvers() static method
+spec: https://tc39.es/proposal-promise-with-resolvers/#sec-promise.withResolvers
+status:
+  baseline: false
+  # baseline_low_date: 2024-<Safari 17.4 release date>
+  support:
+    chrome: "119"
+    chrome_android: "119"
+    edge: "119"
+    firefox: "121"
+    firefox_android: "121"
+    safari: "17.4"
+    safari_ios: "17.4"
```

Comment:
I assume you're doing these by hand? My code for generating these does not include pre-releases. We should probably codify that somewhere (or revisit that decision). For consistency, let's leave it out here (though this will be short lived.)

```suggestion
```

Reply:
Yeah, my little script just copies whatever it sees in BCD for the baseline browsers, it would even copy in "preview".
(I'm not even sure web-features should copy these things from BCD. Probably we should copy this at build time only, but then we need to decide about things like this as well indeed).